pkg/antctl/antctl.go

```diff
@@ -140,7 +142,13 @@ var CommandList = &commandList{
 							usage:     "Get NetworkPolicies applied to the Pod. If present, Namespace must be provided.",
 							shorthand: "p",
 						},
+						{
+							name:      "reference",
+							usage:     "Get NetworkPolicies with specific reference. Reference means its source network policy: K8sNP, ACNP, ANP",
```

Comment:
Reference includes type, name, namespace and UID, should it use `type` and `t` to be more specific?

Reply:
I also wanted to use `-t` at the beginning, but `-t` is a global flag which is `--timeout`. 
I know what you mean. How about changing the name to `referenceType`? Or if you have any other suggestions? Thx.

Reply:
then maybe using "-type" and "-T" for simplicity.

Reply:
Got it.